openfisca_aotearoa/tests/individual_income_tax_rate.yaml

```diff
@@ -0,0 +1,45 @@
+# We can run this test on our command line using `openfisca-run-test openfisca_aotearoa/tests/rates_rebates.yaml`
```

Comment:
Misleading comment. I suggest to remove them all, or at least to remove the filename in it: contributors will be expected to know how to run the tests. At worst, `make test` will do, it will just be slower.